changelogs/current.yaml

```diff
@@ -53,6 +53,10 @@ new_features:
   change: |
     Added support for outlier detection in UDP proxy. This change can be temporarily reverted by setting runtime guard
     ``envoy.reloadable_features.enable_udp_proxy_outlier_detection`` to ``false``.
+- area: dfp
+  change: |
+    Added a knob for disabling the DFP resolve timeout by setting ``envoy.enable_dfp_resolve_timeout`` flag to ``false``.
```

Comment:
does the resolver use dns_query_timeout?  If not I'd argue setting to 0 for "do not run" is slightly better than adding a knob to ignore existing config